tldw_Server_API/app/main.py

```
@@ -5927,6 +5928,10 @@ async def api_metrics():
     except _IMPORT_EXCEPTIONS as _audit_min_err:
         logger.debug(f"Skipping audit router in minimal test app: {_audit_min_err}")
     # Config info endpoints (includes /api/v1/config/jobs used by OpenAPI tests)
+    try:
+        app.include_router(setup_router, prefix=f"{API_V1_PREFIX}", tags=["setup"])
+    except _IMPORT_EXCEPTIONS as _setup_min_err:
+        logger.debug(f"Skipping setup router in minimal test app: {_setup_min_err}")
```

Ruff (0.15.6)

[warning] Logging statement uses f-string

(G004)